### PR TITLE
PF-564 Restore public accessors in QueueMessage files

### PR DESCRIPTION
**Pull request by ddietterich**
Serializing doesn't work if Jackson cannot see the data...

I added a unit test for the case I also found an ordering dependency in the connected tests. Fixed by separating the tests into separate modules.

### CODE REVIEW

**Review by yonghaoy — APPROVED** [review]
Thanks!

**Comment by yonghaoy** [comment]
And I think Tova is out this week

**Comment by yonghaoy** [comment]
Also, one thing that might be help for debuging, but may not safe is printing the message that are failed to serialize.
https://github.com/DataBiosphere/stairway/blob/develop/src/main/java/bio/terra/stairway/QueueMessage.java#L34

File: src/main/java/bio/terra/stairway/QueueMessageNoFields.java [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 // We use this class to read the type information, to guide deserializing the right type
 @JsonIgnoreProperties(ignoreUnknown = true)
-class QueueMessageNoFields extends QueueMessage {
+public class QueueMessageNoFields extends QueueMessage {
```
Comment (pshapiro4broad) [inline src/main/java/bio/terra/stairway/QueueMessageNoFields.java]: I'm pretty sure the class doesn't need to be public, just the fields.

**Review by yonghaoy — APPROVED** [review]

**Review by jmczerk — APPROVED** [review]

**Pull request merged**